content/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -756,6 +798,32 @@ global:
     -----END CERTIFICATE-----
 ```
 
+CA certificates can also be stored in a ConfigMap:
+```yaml
+global:
+  caCertsConfigMap: custom-ca-certs
+```
+
+The ConfigMap must look like this:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: custom-ca-certs
+data:
+  ca-cert1.crt: |
```

Comment:
Do we have any best practices around compound certs? For example, they can become unwieldy and poor performing in a clustered environment with many nodes, if they share a compound certificate.